lib/src/components/NxNavigationSidebar/NxNavigationSidebar.scss

```diff
@@ -0,0 +1,153 @@
+
+$nx-sidebar-text-color: $nx-white;
+$nx-sidebar-background-color: $nx-blue-1000;
+
+$nx-sidebar-active-color: $nx-indigo-400;
+$nx-sidebar-selected-color: $nx-indigo-800;
+$nx-sidebar-hover-color: $nx-indigo-600;
+
+$nx-sidebar-focus-border: 1px solid $nx-focus-border-color;
+
+$nx-sidebar-separator-color: $nx-grey-900;
+
+.nx-page-sidebar-operable {
+  @include container-vertical;
+
+  background-color: $nx-sidebar-background-color;
+  color: $nx-sidebar-text-color;
+  font-size: $nx-font-size;
+  max-width: $nx-sidebar-max-width;
+  padding: $nx-spacing-l 0;
+  width: $nx-sidebar-width__open;
+
+  .nx-text-link {
+    color: $nx-sidebar-text-color;
+  }
+}
+
+.nx-page-sidebar-operable__header {
+  @include container-horizontal;
+
+  display: flex;
+  padding: $nx-spacing-sm $nx-spacing-l $nx-spacing-xl $nx-spacing-l;
+}
+
+.nx-page-sidebar-operable__toggle {
+  border-color: transparent;
+  border-radius: 50%;
+  color: $nx-sidebar-text-color;
+  flex: none;
+  height: 40px;
+  margin: 0;
+  width: 40px;
+
+  &:focus {
+    border: $nx-sidebar-focus-border;
+  }
+
+  &:active {
+    background-color: $nx-sidebar-active-color;
+  }
+
+  &:hover {
+    background-color: $nx-sidebar-hover-color;
+  }
+}
+
+.nx-page-sidebar-operable__product-info {
+  @include container-horizontal;
+
+  align-items: center;
+  color: $nx-sidebar-text-color;
+  display: flex;
+  flex: 1 1 auto;
+  text-decoration: none;
+}
+
+.nx-page-sidebar-operable__logo {
+  height: $nx-sidebar-logo-height;
+  margin-right: $nx-spacing-xs;
+}
+
+.nx-page-sidebar-operable__content {
+  color: $nx-sidebar-text-color;
+  margin: $nx-spacing-l $nx-spacing-md;
+}
+
+.nx-sidebar-navigation {
+  border-bottom: 1px solid $nx-sidebar-separator-color;
+  flex: none;
+  margin-bottom: $nx-spacing-md;
+  padding-bottom: $nx-spacing-xxl;
+}
+
+.nx-sidebar-navigation__link {
+  @include container-horizontal;
+
+  align-items: center;
+  border-color: transparent;
+  display: flex;
+  padding: 0 $nx-spacing-l;
+
+  .nx-icon {
+    margin: $nx-spacing-xs $nx-spacing-xs $nx-spacing-xs 0;
+  }
+
+  &:focus {
+    border: $nx-sidebar-focus-border;
+    border-style: solid none; // only top and bottom borders
+    box-shadow: $nx-focus-box-shadow;
+    outline: none;
+  }
+
+  &:hover {
+    background-color: $nx-sidebar-hover-color;
+  }
+
+  &:active {
+    background-color: $nx-sidebar-active-color;
+  }
+
+  &.selected {
+    @include semi-bold;
+    background-color: $nx-sidebar-selected-color;
+  }
+}
+
+.nx-sidebar-navigation__text {
+  @include nx-truncate-ellipsis;
+}
+
+.open {
+  .nx-page-sidebar-operable__collapsed-content {
+    display: none;
+  }
+}
+
+.closed {
+  width: $nx-sidebar-width__closed;
+  max-width: $nx-sidebar-width__closed;
+
+  .nx-page-sidebar-operable__expanded-content {
+    display: none;
+  }
+
+  .nx-page-sidebar-operable__header {
+    justify-content: center;
+    padding: $nx-spacing-sm $nx-spacing-sm $nx-spacing-xl $nx-spacing-sm;
```

Comment:
Looks like this is only changing the left and right, so it would be preferable to only specify `padding-left` and `padding-right` here and let the top/bottom values fall through from the non-closed styles.

Reply:
> Looks like this is only changing the left and right, so it would be preferable to only specify `padding-left` and `padding-right` here and let the top/bottom values fall through from the non-closed styles.

Should be fixed by d72ab4b12858cfa31a60a1644b2ec603574bef31